version/version.go

```diff
@@ -22,7 +22,7 @@ var (
 
 	// Version is the main version number that is being run at the moment.
 	// Note: our current release process does a pattern match on this variable.
-	Version = "0.5.0"
+	Version = "0.6.0"
```

Comment:
Thanks for thinking of this! We actually bump this version as part of our release process, so it's not ready to be changed yet. Changing it now may cause some confusion later. Would you mind reverting this change?